data/update.py

```diff
@@ -43,38 +45,65 @@
 # domain-scan information
 SCAN_TARGET = os.path.join(this_dir, "./output/scan")
 SCAN_COMMAND = os.environ.get("DOMAIN_SCAN_PATH", None)
-SCANNERS = os.environ.get("SCANNERS", "inspect,tls,analytics")
-ANALYTICS_URL = "https://analytics.usa.gov/data/live/second-level-domains.csv"
-
-# TODO:
-# --date: override date
-# --skip-scan: skip the scanning part (rely on local scan data)
-# --skip-upload: don't bother uploading anything to S3
-
-def run():
+SCANNERS = os.environ.get("SCANNERS", "inspect,tls,analytics,sslyze")
```

Comment:
This re-adds `sslyze` to the list of scanners run in production. We don't surface this data in Pulse directly yet, but it's very good to be collecting it along the way..